package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "todolist-v2",
+  "version": "1.0.0",
+  "description": "> Build a To Do List JavaScript application",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eapenzacharias/todolist-v2.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/eapenzacharias/todolist-v2/issues"
+  },
+  "homepage": "https://github.com/eapenzacharias/todolist-v2#readme",
+  "devDependencies": {
```

Comment:
- [ ] The _dervServer_ package is missing your application. Kindly install the same and add that in your **webpack** configuration, by running the command  `npm install -D webpack-dev-server`
to enable your application to be run on the server locally.

**devServer in webpack configuration file:**
![image](https://user-images.githubusercontent.com/12745474/134922749-d0016f56-002e-4415-8882-4eb3dcbf6551.png)